backend/SyncReverseProxy/HgRequestTransformer.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Text.RegularExpressions;
+using Yarp.ReverseProxy.Forwarder;
+
+namespace LexSyncReverseProxy;
+
+public partial class HgRequestTransformer : HttpTransformer
+{
+    [GeneratedRegex(@"^/hg/[a-z-0-9]/")]
```

Comment:
This might need to have the `-` removed, I didn't have this file around when I was doing #654.